apps/docs/content/components/concepts/responsive-styles.mdx

```diff
@@ -30,12 +32,12 @@ By default, Hopper uses a mobile-first breakpoint system, similar to what you mi
 
 <Example src="HopperProvider/docs/responsive-styles/mobile-first-do" isOpen />
 
-For this reason, it's often a good idea to implement the mobile layout for a design first, then layer on any changes that make sense for sm screens, followed by md screens, etc.
+It's often a good idea to implement the mobile layout for a design first, then gradually add styles as the screen size increases and the design requires, progressing from small screens to wider screens.
```

Comment:
This sentence seems a bit weird to me.. specifically the part that says "and the design requires". Maybe we could simplify to something like this:
```suggestion
It’s often best to start with the mobile first layout for a design, then add styles as the screen size increases, moving from small to larger screens.
```